docs/csharp/programming-guide/classes-and-structs/nested-types.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Nested Types - C# Programming Guide"
+description: A type defined within a class, struct, or interface is called a nested type in C#. Nested types are accessible only from their containing type.
```

Comment:
The last sentence is incorrect. (If they have the default, `private`, access, it's true.)

```suggestion
description: A type defined within a class, struct, or interface is called a nested type in C#. 
```